### PR TITLE
Update Application.scala

### PR DESCRIPTION
**Pull request by Doedels**
Hello Dan

### CODE REVIEW

File: src/main/scala/com/danielyli/gitplayground/Application.scala [hunk 1]
```diff
@@ -2,7 +2,7 @@ package com.danielyli.gitplayground
 
 object Application {
   def main(args: Array[String]): Unit = {
-    println("Hello, world!")
+    println("Hello, Dan!")
```
Comment (danielyli) [inline src/main/scala/com/danielyli/gitplayground/Application.scala]: Thanks, Sanders, but I think the exclamation is not needed. Go ahead and change it to a period please.

File: src/main/scala/com/danielyli/gitplayground/Application.scala [hunk 2]
```diff
@@ -2,7 +2,7 @@ package com.danielyli.gitplayground
 
 object Application {
   def main(args: Array[String]): Unit = {
```
Comment (danielyli) [inline src/main/scala/com/danielyli/gitplayground/Application.scala]: Let’s change `args` to `arguments`.

File: src/main/scala/com/danielyli/gitplayground/Application.scala (hunk 1 above)
Comment (Doedels) [reply src/main/scala/com/danielyli/gitplayground/Application.scala]: Ok, I can live with that. I'l change it.

Comment (danielyli) [reply src/main/scala/com/danielyli/gitplayground/Application.scala]: Thanks

File: src/main/scala/com/danielyli/gitplayground/Application.scala (hunk 2 above)
Comment (danielyli) [reply src/main/scala/com/danielyli/gitplayground/Application.scala]: Ok, I can accept `args`

**Pull request merged**